rfcs/20191122-tf-build-tlc.md

```diff
@@ -0,0 +1,213 @@
+# Tensorflow Build TLC
+
+| Status        | Proposed                         |
+| :------------ | :------------------------------- |
+| **Author(s)** | Brian Zhao (bmzhao@google.com)   |
+| **Sponsor**   | Gunhan Gulsoy (gunan@google.com) |
+| **Updated**   | 2019-11-22                       |
+
+## Objective
+
+Simplify Tensorflow’s build so that
+1. Builds are more granular
+2. Builds are easier to reason about
+3. TF’s build is well positioned to immediately benefit from [Bazel shared library support](https://docs.google.com/document/d/1qYHGcxP9BEi7t9b53r8STr9G3XVtvwy9V-EXqr54Xxg/edit#)
```

Comment:
Please link to a public document on this feature, or remove it.

Reply:
looks like this is the one https://docs.google.com/document/d/13nolQXvXDsJ4yjWP1FRd-WscP2-rHZcR3Zb__js6mLA/edit#heading=h.7795zogmlqtv

Reply:
Done (although the original doc link was a public doc).